### DISCUSSION TITLE
does this need complete jQuery?

### DISCUSSION THREAD

**Discussion by Rajaniraiyn** [Tronic247/material#45] [Q&A] [answered]
## Hi Guys!

This is a awesome project.

I understand that this **material**  is dependent on jQuery.

I replaced the [jQuery](https://code.jquery.com/jquery-3.6.0.min.js) with j[Query Slim](https://code.jquery.com/jquery-3.6.0.min.js). [here](http://jsfiddle.net/vpg6wudm/1/) everything seems to work perfectly.

If this only requires jQuery Slim then it will be awesome as it is lightweight as compared to jQuery.

I just want to confirm that.

**Comment by Posandu** [comment]
Great news ! You don't need jquery anymore

**Reply by Posandu** [reply]
And please note that you can join us on [https://discord.gg/nb2xXcRF](https://discord.gg/nb2xXcRF)

**Reply by Rajaniraiyn** [reply]
then please change the documentation and others. it will be very useful.

sorry I'm not in discord.
I just asked this for some conversion.
If I got time I will port this project to vanilla JS

**Comment by Posandu** [comment, marked as answer]
@Rajaniraiyn This project is now pure JS
